src/design-tokens/themes.json

```diff
@@ -550,13 +561,13 @@
               },
               "mediumEmphasis": {
                 "@": {
-                  "value": "{eds.color.neutral.450}"
+                  "value": "{eds.color.neutral.550}"
                 },
                 "hover": {
-                  "value": "{eds.color.neutral.550}"
+                  "value": "{eds.color.neutral.650}"
                 },
                 "active": {
-                  "value": "{eds.color.neutral.650}"
```

Comment:
Snapshot changes come from these three changes. Any components using medium emphasis tokens will be accepted.